perun-ldapc/src/main/java/cz/metacentrum/perun/ldapc/model/impl/PerunResourceImpl.java

```diff
@@ -76,13 +77,15 @@
 				);
 	}
 
-	public void addResource(Resource resource, String entityID) throws InternalErrorException {
+	public void addResource(Resource resource, Map<String,String> facilityAttributes) throws InternalErrorException {
```

Comment:
I suggest naming the second parameter only attributes - it is more generic. Also use list of PerunAttribute objects, not string values.

Reply:
I removed the second param all together.